modules/installing-microshift-rpm.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc 
+
+:_content-type: PROCEDURE
+[id="installing-microshift-from-rpm-package_{context}"]
+= Installing MicroShift from an RPM package 
+
+Use the following procedure to install MicroShift from an RPM package. 
+
+.Prerequisites 
+
+* The xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[system requirements for installing MicroShift] have been met. 
+* You have completed the steps at xref:../microshift_install/microshift-install-rpm.adoc#preparing-install-microshift-from-rpm-package_microshift-install-rpm[Preparing your system for installing MicroShift from an RPM package].
+
+.Procedure
+
+. Enable the MicroShift repositories by entering the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos \
+    --enable rhocp-4.12-for-rhel-8-$(uname -i)-rpms \
```

Comment:
This name will be version-specific. We don't have to deal with that now, but is there a standard way to ensure that we refer to the right version of OCP and RHEL here? Maybe using a variable like we do with some of the other names and versions?

Reply:
Done.